oauth2client/file.py

```diff
@@ -20,17 +20,28 @@
 
 import os
 import threading
+import warnings
 
 from oauth2client import client
 
 
 __author__ = 'jcgregorio@google.com (Joe Gregorio)'
 
+_SYM_LINK_MESSAGE = 'File: {0}: Is a symbolic link.'
+
+_IS_DIR_MESSAGE = '{0}: Is a directory'
+
+_MISSING_FILE_MESSAGE = 'Cannot access {0}: No such file or directory'
+
 
 class CredentialsFileSymbolicLinkError(Exception):
     """Credentials files must not be symbolic links."""
 
 
+class CredentialsFileDirectoryError(Exception):
```

Comment:
I was just trying to be consistent within the file...  IOError sounds reasonable to me.

If I use IOError, and we remove the Symbolic Link parts, then there will only be 2 real Exceptions defined outside of `client` (in the core library).